python/grass/jupyter/timeseriesmap.py

```diff
@@ -137,8 +120,8 @@ class TimeSeriesMap:
 
     >>> img = TimeSeriesMap("series_name")
     >>> img.d_legend()  # Add legend
-    >>> img.time_slider()  # Create TimeSlider
-    >>> img.animate()
+    >>> img.show()  # Create TimeSlider
+    >>> img.save()
```

Comment:
Unlike animate(), save() does not make sense without parameter because it would go to a temporary directory.